### PR TITLE
Add kiwisolver 0.1.3 and atom 0.3.9

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
@sccolbert @blink1073 @MatthieuDartiailh I am adding kiwisolver and atom (and enaml after these two are in) to conda-forge for linux, mac and windows. This will allow users to easily install this with [conda] (http://conda.pydata.org/docs/) (conda install -c conda-forge atom) (conda install -c conda-forge kiwisolver). The maintenance burden here is light as all testing and releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/atom-feedstock and https://github.com/conda-forge/kiwisolver-feedstock (and the other packages as <package>-feedstock) shortly after this PR is finished and merged. If you would like to be added as a maintainer (to make new releases for instance), I can happily add you to the maintainers list. If not, that is totally fine too!  Is there anyone else that you think would want to be pinged about maintaining this build?

**Comment by MatthieuDartiailh** [comment]
That's great. I am building conda packages for my forks compatible with python 3 (you can check it out on ecpy channel) but having up to date package of master is nice too. You can add me as a maintainer.

**Comment by ericdill** [comment]
Yes thank you so very much for your work on making the enaml stack python 3 compatible!  I have conda packages for your forks too [here] (https://github.com/NSLS-II/auto-build-tagged-recipes/tree/master/enaml-stack) and will update these conda-forge recipes as soon as they are merged into the upstream repos.  I'll add you as a maintainer now.

**Comment by blink1073** [comment]
Hi @ericdill, you can add me as well.

**Comment by sccolbert** [comment]
You can add me as well. Thanks so much for doing this!

**Comment by ericdill** [comment]
@MatthieuDartiailh @blink1073 @sccolbert Added you all as maintainers. Thanks for responding!  I assume you would also like to be added as maintainers of enaml?  After this PR is merged and the atom/kiwisolver packages are built and available on anaconda.org/conda-forge I'll submit a PR for enaml here on conda-forge.

How is the py3 compatibility coming along?

File: recipes/atom/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - atom
+    - atom.datastructures
+    - tests
+
+about:
+  home: https://github.com/nucleic/atom
+  license: Modified BSD
```
Comment (ocefpaf) [inline recipes/atom/meta.yaml]: Modified BSD is the `BSD 3-Clause`, right?

Comment (ericdill) [inline recipes/atom/meta.yaml]: Wikipedia suggests yes: https://en.wikipedia.org/wiki/BSD_licenses#3-clause_license_.28.22Revised_BSD_License.22.2C_.22New_BSD_License.22.2C_or_.22Modified_BSD_License.22.29

Comment (ocefpaf) [inline recipes/atom/meta.yaml]: Yeah. That was I got my info too :wink:

**Comment by blink1073** [comment]
Yep, you can add me to enaml as well.

**Comment by MatthieuDartiailh** [comment]
Python 3 compatibility is waiting for review (at least from my point of view, as as far as I can tell it works pretty well).

File: recipes/kiwisolver/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+package:
+  name: kiwisolver
+  version: {{ version }}
+
+source:
+  fn: kiwisolver-{{ version }}.tar.gz
+  url: https://github.com/nucleic/kiwi/archive/{{ version }}.tar.gz
+  sha256: 761757e93f884223281af7969df85a93542a66769ca13617084baf82848c656d
+  # patches:
+  #   - setup.py.patch
```
Comment (ocefpaf) [inline recipes/kiwisolver/meta.yaml]: @ericdill is this good to go or are you adding a patch here? (If not let's remove the comments and merge as-is)

Comment (ericdill) [inline recipes/kiwisolver/meta.yaml]: This was a patch that nuked the install_requires bit in setup.py, but I apparently already figured out that I didnt need it, or something.  Anyway, removing this comment and then should be good to go. Thanks!

**Comment by jakirkham** [comment]
LGTM. Thanks @ericdill.

**Pull request merged**